### PR TITLE
feat_BE_networkSpeedDebug

### PR DESCRIPTION
**Pull request by Trip4077**
# Description

Please include a summary of the change and a link to which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change status
- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: backend/child_processes/networkApiChild.js [hunk 1]
```diff
@@ -1,9 +1,19 @@
...
+    const test = speedTest({ maxTime: 1000, serverId: selectServer(msg.state)});
+    
+    test.on('config', config => {
+        //console.log('Configuration info:');
+        //console.dir(config);
+      });
+  
+      test.on('data', data => {
+        //console.log('data info: ')
+        console.dir(data)
```
Comment (CLantigua2) [inline backend/child_processes/networkApiChild.js]: clean up unneeded console commands

File: backend/child_processes/stateMap.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+            if(states[i][0] == input){
+                return(states[i][1]);
+            }
+        }    
+    } else if (to == 'name'){
+        input = input.toUpperCase();
+        for(i = 0; i < states.length; i++){
+            if(states[i][1] == input){
+                return(states[i][0]);
+            }
```
Comment (CLantigua2) [inline backend/child_processes/stateMap.js]: please make line 56 readable by adding proper spacing and include a `let` keyword in the for loop to not throw off any other variables that might be on the file with the same name. Currently it is defaulted to a var which does not follow scoping rules.

File: backend/api/routes/networkRoutes/networkRoutes.js [hunk 3]
```diff
@@ -4,15 +4,16 @@ const router = express.Router();
 
 // url/api/network
 router.get('/', (req, res) => {
-    const forked = fork('./backend/child_processes/networkApiChild.js');
+        console.log("REQUEST", req.query.state)
```
[the remark this answers is not in the document]
Comment (Trip4077) [reply backend/api/routes/networkRoutes/networkRoutes.js]: Nope, just missed this one

File: backend/child_processes/networkApiChild.js (hunk 1 above)
Comment (Trip4077) [reply backend/child_processes/networkApiChild.js]: All gone 😅

**Pull request merged**